src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+/**
+ * A class that handles the user interface of the Duke bot.
+ */
+public class Ui {
```

Comment:
Maybe this UI class could handle displaying of all messages instead of just welcome and goodbye